test/e2e/framework/util.go

```diff
@@ -3219,7 +3219,17 @@ func IssueSSHCommandWithResult(cmd, provider string, node *v1.Node) (*SSHResult,
 	}
 
 	if host == "" {
-		return nil, fmt.Errorf("couldn't find external IP address for node %s", node.Name)
+		// No external IPs were found, let's try to use internal as plan B.
```

Comment:
Could you please provide a reference document about deprecation. I checked this doc and nothing says that node's internal IP is deprecated.